lib/std/hash/RapidHash.zig

```diff
@@ -0,0 +1,125 @@
+const std = @import("std");
+
+const readInt = std.mem.readInt;
+const assert = std.debug.assert;
+const expect = std.testing.expect;
+const expectEqual = std.testing.expectEqual;
+
+const RAPID_SEED: u64 = 0xbdd89aa982704029;
+const RAPID_SECRET: [3]u64 = .{ 0x2d358dccaa6c78a5, 0x8bb84b93962eacc9, 0x4b33a62ed433d4a3 };
+
+pub fn hash(seed: u64, input: []const u8) u64 {
+    const sc = RAPID_SECRET;
+    const len = input.len;
+    var a: u64 = 0;
+    var b: u64 = 0;
+    var k = input;
+    var is: [3]u64 = .{ seed, 0, 0 };
```

Comment:
Would you be able to name these variables something more descriptive? The code is a bit hard to read right now since it's difficult to understand what each variable is representing.

Reply:
Ya for sure, that was mostly to ease writing it since it was easier for me to read, I'll do some rename swaps on those.